crates/snark-acct-types/src/messages.rs

```diff
@@ -9,14 +9,14 @@ pub struct MessageEntry {
     source: AccountId,
 
     /// The epoch that the message was included.
-    incl_epoch: u32,
+    incl_epoch: u64,
 
     /// The message payload.
     payload: MsgPayload,
 }
 
 impl MessageEntry {
-    pub fn new(source: AccountId, incl_epoch: u32, payload: MsgPayload) -> Self {
+    pub fn new(source: AccountId, incl_epoch: u64, payload: MsgPayload) -> Self {
```

Comment:
`u32` or actually the `Epoch` type exists now.

Reply:
With slots being u64, I think epoch should be u64 as well. See another comment.